routing/custom_route_loader.rst

```diff
@@ -119,14 +119,20 @@ and configure the service and method to call:
         namespace Symfony\Component\Routing\Loader\Configurator;
 
         return function (RoutingConfigurator $routes) {
-            $routes->import('admin_route_loader:loadRoutes', 'service');
+            $routes->import('admin_route_loader::loadRoutes', 'service');
         };
 
 In this example, the routes are loaded by calling the ``loadRoutes()`` method
 of the service whose ID is ``admin_route_loader``. Your service doesn't have to
 extend or implement any special class, but the called method must return a
 :class:`Symfony\\Component\\Routing\\RouteCollection` object.
 
+If you're using :ref:`autoconfigure <services-autoconfigure>`, your service should
+implement the `ServiceRouterLoaderInterface` interface to be tagged automatically by
+Symfony!
```

Comment:
I think we can make this less expressive and replace `!` with `.`. Can you also add the next sentence directly to this paragraph, to save some space?

Reply:
I would remove the “by Symfony!” Completely and end with a dot